sdk/corevm_guest.h

```diff
@@ -33,25 +33,15 @@ POLKAVM_IMPORT(uint64_t, corevm_yield_audio_frame_impl, uint64_t, uint64_t, uint
                 n = COREVM_PRINTF_BUFFER_LEN - 1; \
             } \
             buffer[n] = 0; \
-            while (true) { \
-                uint64_t ret = corevm_yield_console_data(stream, (uint64_t)buffer, (uint64_t)(n + 1)); \
-                if (ret == 0) { \
-                    break; \
-                } \
-            } \
+            corevm_yield_console_data(stream, (uint64_t)buffer, (uint64_t)(n + 1)); \
         } \
     }
 
 #define corevm_printf(format, ...) corevm_printf_impl(1, format, ##__VA_ARGS__)
 #define corevm_eprintf(format, ...) corevm_printf_impl(2, format, ##__VA_ARGS__)
```

Comment:
You should be able to delete these now completely since stdio can now go through normal Linux syscalls, no?

Reply:
Sure. Removed them.